documentation/api/wire_format/deactivate_node_format_v3.markdown

```diff
@@ -0,0 +1,60 @@
+---
+title: "PuppetDB 3.0 » API » Deactivate Node Wire Format, Version 3"
+layout: default
+canonical: "/puppetdb/latest/api/wire_format/deactivate_node_format_v3.html"
+---
+
+PuppetDB receives deactivate node commands from puppet masters in the following wire format. 
```

Comment:
I think "Puppet masters" is our preferred capitalization per https://github.com/puppetlabs/puppet-docs/blob/master/style_and_usage.markdown#other-capitalization-questions